crates/redis/src/lib.rs

```diff
@@ -95,7 +95,13 @@ impl RedisTrigger {
         loop {
             match stream.next().await {
                 Some(msg) => drop(self.handle(msg).await),
-                None => log::trace!("Empty message"),
+                None => {
+                    log::trace!("Empty message");
+                    if !client.check_connection() {
+                        log::trace!("No Redis connection available");
```

Comment:
Should this be an `info!` log, similar to the one informing about the subscription?